core/src/main/java/de/bwaldvogel/mongo/backend/AbstractMongoCollection.java

```diff
@@ -33,6 +31,8 @@
 import de.bwaldvogel.mongo.exception.MongoServerException;
 import de.bwaldvogel.mongo.oplog.Oplog;
 
+import static de.bwaldvogel.mongo.backend.Constants.ID_FIELD;
```

Comment:
to avoid an "organize import" edit war, please apply the project-specific code style from the [`code-style/`](https://github.com/bwaldvogel/mongo-java-server/tree/master/code-style) directory.

Reply:
Fixed, thanks!